climateset/download/downloader.py

```diff
@@ -933,6 +906,41 @@ def download_raw_input(self):
                 self.download_raw_input_single_var(v, institution_id="IAMC", save_to_meta=True)
 
 
+def filter_download_script(wget_script_content, start_year, end_year):
```

Comment:
Sorry about this one; there are not a lot of options for dealing with this wget script. I opted for a stateful regex algorithm.